### PR TITLE
API and Postman 💖

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

This PR reference the Postman collection in the API overview

This PR enhance our Postman collection:

* Standardise endpoint description and titles
* Create example when it's relevant
* Create placeholder for required parameter at minimum

Extra

* Fixes Main doc api desc that are wrong
* Minor typo and layout update

### Motivation

1. Create an OpenAPI 3.0 version of our bash doc.
2. Enhance customer on-boarding on DD API
3. [API endpoints always need some love](https://www.youtube.com/watch?v=WpYeekQkAdc)

### Preview link
https://docs-staging.datadoghq.com/gus/api-update/api

To review the postman collection: https://documenter.getpostman.com/view/7156598/S1LpbsRV

### CODE REVIEW

File: content/en/api/overview/overview.md [hunk 1]
```diff
@@ -5,10 +5,13 @@ order: 1
 external_redirect: /api/#overview
 ---
 ## Overview
-This section details Datadogs HTTP API. It makes it easy to get data in and out of Datadog.
+This section details Datadog's HTTP API. It makes it easy to get data in and out of Datadog.
```
Comment (ruthnaebeck) [inline content/en/api/overview/overview.md]:
```suggestion
This section details Datadog's HTTP API for getting data in and out of Datadog.
```

File: content/en/api/overview/overview.md [hunk 2]
```diff
@@ -5,10 +5,13 @@ order: 1
...
+
+**In order to get started on Datadog HTTP API, use our [Datadog Postman collection][1]**
```
Comment (ruthnaebeck) [inline content/en/api/overview/overview.md]:
```suggestion
**To get started on Datadog HTTP API, use our [Datadog Postman collection][1]**
```

File: content/en/api/synthetics/post_test.md [hunk 3]
```diff
@@ -36,15 +36,15 @@ A browser test is treated like a GET API test. This method gives you the ability
...
     <td><code>contains</code>, <code>does not contain</code>, <code>is</code>, <code>is not</code>, <code>matches</code>, <code>does not match</code></td>
-    <td>for <code>contains</code>/<code>does not contain</code>/<code>is</code>/<code>is not</code>: String for <code>matches</code>/<code>does not match</code>: [RegexStrings][3]</td>
+    <td>for <code>contains</code>/<code>does not contain</code>/<code>is</code>/<code>is not</code>: String for <code>matches</code>/<code>does not match</code>: <a href="https://docs.datadoghq.com/tagging/using_tags">RegexString</a></td>
   </tr>
   <tr>
     <td>Body</td>
     <td><code>contains</code>, <code>does not contain</code>, <code>is</code>, <code>is not</code>, <code>matches</code>, <code>does not match</code></td>
-    <td>for <code>contains</code>/<code>does not contain/<code>is</code>/<code>is not</code>: String for <code>matches</code>/<code>does not match</code>: [RegexStrings][3]</td>
+    <td>for <code>contains</code>/<code>does not contain/<code>is</code>/<code>is not</code>: String for <code>matches</code>/<code>does not match</code>: <a href="https://docs.datadoghq.com/tagging/using_tags">RegexString</a></td>
```
Comment (ruthnaebeck) [inline content/en/api/synthetics/post_test.md]:
```suggestion
    <td>for <code>contains</code>/<code>does not contain</code>/<code>is</code>/<code>is not</code>: String for <code>matches</code>/<code>does not match</code>: <a href="https://docs.datadoghq.com/tagging/using_tags">RegexString</a></td>
```

File: content/en/api/synthetics/put_test.md [hunk 4]
```diff
@@ -40,15 +40,15 @@ In order to update a request, you have to submit a full object, but only these p
...
     <td>Headers</td>
     <td><code>contains</code>, <code>does not contain</code>, <code>is</code>, <code>is not</code>, <code>matches</code>, <code>does not match</code></td>
-    <td>for <code>contains</code>/<code>does not contain</code>/<code>is</code>/<code>is not</code>: String for <code>matches</code>/<code>does not match</code>: RegexStrings</td>
+    <td>for <code>contains</code>/<code>does not contain</code>/<code>is</code>/<code>is not</code>: String for <code>matches</code>/<code>does not match</code>: <a href="https://docs.datadoghq.com/tagging/using_tags">RegexString</a></td>
   </tr>
   <tr>
     <td>Body</td>
     <td><code>contains</code>, <code>does not contain</code>, <code>is</code>, <code>is not</code>, <code>matches</code>, <code>does not match</code></td>
-    <td>for <code>contains</code>/<code>does not contain/<code>is</code>/<code>is not</code>: String for <code>matches</code>/<code>does not match</code>: RegexStrings</td>
+    <td>for <code>contains</code>/<code>does not contain/<code>is</code>/<code>is not</code>: String for <code>matches</code>/<code>does not match</code>: <a href="https://docs.datadoghq.com/tagging/using_tags">RegexString</a></td>
```
Comment (ruthnaebeck) [inline content/en/api/synthetics/put_test.md]:
```suggestion
    <td>for <code>contains</code>/<code>does not contain</code>/<code>is</code>/<code>is not</code>: String for <code>matches</code>/<code>does not match</code>: <a href="https://docs.datadoghq.com/tagging/using_tags">RegexString</a></td>
```

File: content/en/api/tags/tags_delete.md [hunk 5]
```diff
@@ -9,8 +9,5 @@ external_redirect: /api/#remove-host-tags
 This endpoint allows you to remove all tags in an integration source for a single host.
 
 ##### ARGUMENTS
```
Comment (ruthnaebeck) [inline content/en/api/tags/tags_delete.md]: Not sure how you are suppose to remove all tags from a source unless you specify the source. Either the argument is needed or the description needs to change.

File: static/resources/json/datadog_collection.json [hunk 6]
```diff
@@ -126,111 +289,1052 @@
...
+							},
+							"status": "Bad Request",
+							"code": 400,
+							"_postman_previewlanguage": "json",
+							"header": [
 								{
-									"key": "api_key",
-									"value": "{{dd_api_key}}"
+									"key": "Date",
+									"value": "Fri, 19 Apr 2019 12:38:35 GMT"
```
Comment (ruthnaebeck) [inline static/resources/json/datadog_collection.json]: Is this correct?

File: static/resources/json/datadog_collection.json [hunk 7]
```diff
@@ -296,90 +1515,241 @@
...
 							}
 						],
 						"body": {
 							"mode": "raw",
-							"raw": "{\n    \"title\": \"Sum of Memory Free 2\",\n    \"widgets\": [\n        {\n            \"definition\": {\n                \"type\": \"timeseries\",\n                \"requests\": [\n                    {\n                        \"q\": \"sum:system.mem.free{*}\"\n                    }\n                ],\n                \"title\": \"Sum Memory Free Shell\"\n            }\n        }\n    ],\n    \"layout_type\": \"ordered\",\n    \"description\": \"An updated dashboard with memory info.\",\n    \"is_read_only\": true,\n    \"notify_list\": [\n        \"test@datadoghq.com\"\n    ],\n    \"template_variables\": [\n        {\n            \"name\": \"host1\",\n            \"prefix\": \"host\",\n            \"default\": \"myhost\"\n        }\n    ]\n}"
+							"raw": ""
 						},
 						"url": {
-							"raw": "https://api.datadoghq.com/api/v1/dashboard/:DASHBOARD_ID?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
+							"raw": "https://app.datadoghq.com/api/v1/dashboard/?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
```
Comment (ruthnaebeck) [inline static/resources/json/datadog_collection.json]: It's `app` here and `api` below. Looks like `api` is correct.

File: content/en/api/embeds/embeds_get.md [hunk 8]
```diff
@@ -21,13 +21,5 @@ Returns: A JSON object with 8 elements:
...
-    * **small**, 
-    * **medium**, 
-    * **large**, 
-    * **xlarge**.
-* **`legend`** [*optional*, *default*=**no**]:  
-    The flag determining if the graph includes a legend. Must be one of **yes** or **no**.
-* **`template_variables`** [*optional*, *default*=**None**]:  
-    Replace template variables in queries with form $var. To replace $var with val, use var=val as a parameter for each template variable you wish to replace. If any template variables are missing values in the iframe source url, then (*) is used as the value.
+
+This endpoint takes no JSON arguments.
```
Comment (ruthnaebeck) [inline content/en/api/embeds/embeds_get.md]: How can you get an specific embed without at least specifying the id?

**Comment by ruthnaebeck** [comment]
Is the json file for loading into Postman? I thought Brendan already updated that.

File: content/en/api/embeds/embeds_get.md (hunk 8 above)
Comment (l0k0ms) [reply content/en/api/embeds/embeds_get.md]: The embed ID is in the URL directly: https://docs.datadoghq.com/api/?lang=python#get-specific-embed

File: content/en/api/tags/tags_delete.md (hunk 5 above)
Comment (l0k0ms) [reply content/en/api/tags/tags_delete.md]: The description is wrong, it should remove tags associated to a host and that's it.

File: content/en/api/tags/tags_delete.md [hunk 9]
```diff
@@ -9,8 +9,5 @@ external_redirect: /api/#remove-host-tags
 This endpoint allows you to remove all tags in an integration source for a single host.
```
Comment (l0k0ms) [inline content/en/api/tags/tags_delete.md]:
```suggestion
This endpoint allows you to remove all tags for a single host.
```

File: static/resources/json/datadog_collection.json (hunk 7 above)
Comment (l0k0ms) [reply static/resources/json/datadog_collection.json]: it should be api indeed, updating the collection in another commit

**Pull request merged**